core/src/main/scala/kafka/utils/KafkaScheduler.scala

```diff
@@ -50,9 +50,10 @@ trait Scheduler {
    * @param name The name of this task
    * @param delay The amount of time to wait before the first execution
    * @param period The period with which to execute the task. If < 0 the task will execute only once.
-   * @param unit The unit for the preceding times.
+   * @param unit The unit for the preceding times
+   * @return the task scheduled
```

Comment:
nit: it's not really the task that was scheduled, but just a future that can be used to manage the task scheduled.